content/en/user-guide/aws/elasticsearch/index.md

```diff
@@ -111,7 +111,7 @@ $ curl http://my-domain.us-east-1.es.localhost.localstack.cloud:4566
 Or the health endpoint:
 
 {{< command >}}
-$ curl -s http://my-domain.us-east-1.es.localhost.localstack.cloud:4566/_cluster/health | jq .
+$ curl -s http://my-domain.us-east-1.es.localhost.localstack.cloud:4566/_cluster/_localstack/health | jq .
```

Comment:
Here it's the same as for opensearch (see below). The URLs here point to the health check of elasticsearch (and not LocalStack). Therefore the URL does not change with the new release.